django/contrib/postgres/aggregates/base.py

```diff
@@ -0,0 +1,8 @@
+from django.db.models import Aggregate
+
+
+class DistinctAggregate(Aggregate):
```

Comment:
Yes, this is a good idea, it's not really postgres specific (other than postgres supporting it in most agg functions). I'll do that now :+1: